### PR TITLE
Distributed analysis words

### PR DESCRIPTION
**Pull request by mesmith75**

### CODE REVIEW

File: glossary/s.md [hunk 1]
```diff
@@ -1 +1,4 @@
 # S
+
+**Storage Element (SE)**
+: A service at a grid site that stores data.
```
Comment (eduardo-rodrigues) [inline glossary/s.md]: Worth giving a couple of examples.

**Comment by eduardo-rodrigues** [comment]
For your information I prepared in https://github.com/lhcb/glossary/pull/5 C&SW definitions for terms that can somewhat relate with distributing computing. Just mentioning to avoid overlaps ...

**Comment by mesmith75** [comment]
I think we have the most important now.

**Comment by eduardo-rodrigues** [comment]
Many thanks for the contributions!

**Comment by henryiii** [comment]
Fixed one more thing, and merged. Looks great, thanks!

**Pull request merged**